tests/rule_based_profiler/bobby_user_workflow_verbose_profiler_config.yml

```diff
@@ -20,43 +23,54 @@ rules:
-        round_to_nearest_integer: True
+        round_decimals: 0
+        truncate_distribution:
+          lower_bound: 0
     expectation_configuration_builders:
       - expectation_type: expect_table_row_count_to_be_between
+        class_name: DefaultExpectationConfigurationBuilder
+        module_name: great_expectations.rule_based_profiler.expectation_configuration_builder
         min_value: $parameter.row_count.value.min_value
         max_value: $parameter.row_count.value.max_value
+        mostly: $variables.mostly
         meta:
           profiler_details: $parameter.row_count.details
-  # TODO: <Alex>ALEX -- this is to be implemented/supported next.</Alex>
-#  ranges_rule:
-#    domain_builder:
-#      class_name: SimpleSemanticTypeColumnDomainBuilder
-#      semantic_types:
-#        - numeric
-#    parameter_builders:
-#      - parameter_name: min
-#        class_name: NumericMetricRangeMultiBatchParameterBuilder
-#        batch_request: $variables.jan_feb_2019_trip_data_batch_request
-#        metric_name: column.min
-#        metric_domain_kwargs: $domain.domain_kwargs
-#        false_positive_rate: $variables.false_positive_rate
-#      - parameter_name: max
-#        class_name: NumericMetricRangeMultiBatchParameterBuilder
-#        batch_request: $variables.jan_feb_2019_trip_data_batch_request
-#        metric_name: column.max
-#        metric_domain_kwargs: $domain.domain_kwargs
-#        false_positive_rate: $variables.false_positive_rate
-#    expectation_configuration_builders:
-#      - expectation_type: expect_column_min_to_be_between
-#        min_value: $parameter.min.value.min_value
-#        max_value: $parameter.min.value.max_value
-#        # TODO: <Alex>ALEX</Alex>
-#        # mostly: $parameter.min.value.mostly # call it something useful
-#        # TODO: <Alex>ALEX</Alex>
-#        meta:
-#          profiler_details: $parameter.min.details
-#      - expectation_type: expect_column_max_to_be_between
-#        min_value: $parameter.max.value.min_value
-#        max_value: $parameter.max.value.max_value
-#        meta:
-#          profiler_details: $parameter.max.details
+  column_ranges_rule:
+    domain_builder:
+      class_name: SimpleSemanticTypeColumnDomainBuilder
+      semantic_types:
+        - numeric
+    parameter_builders:
+      - parameter_name: min
+        class_name: NumericMetricRangeMultiBatchParameterBuilder
+        batch_request: $variables.jan_feb_2019_trip_data_batch_request
+        metric_name: column.min
+        metric_domain_kwargs: $domain.domain_kwargs
+        false_positive_rate: $variables.false_positive_rate
+        round_decimals: 2
+      - parameter_name: max
+        class_name: NumericMetricRangeMultiBatchParameterBuilder
+        batch_request: $variables.jan_feb_2019_trip_data_batch_request
+        metric_name: column.max
+        metric_domain_kwargs: $domain.domain_kwargs
+        false_positive_rate: $variables.false_positive_rate
+        round_decimals: 2
```

Comment:
As commented in the `NumericMetricRangeMultiBatchParameterBuilder`, it seems excessive to require two separate parameter_builders for min and max here, unless we expect that the `NumericMetricRangeMultiBatchParameterBuilder` would be used to build other types of metrics, or that the user might have different specifications for min and max (like different numbers of decimal places).